flask/utils/run.sh

```diff
@@ -16,6 +16,10 @@ elif [[ "$1" == "prod" ]]; then
     $COMMAND db upgrade
     gunicorn wsgi:app "$@"
 else
+    if [ ! -d "migrations" ]; then
+        $COMMAND db init
+        $COMMAND db migrate -m "Initial migration."
+    fi
```

Comment:
Interesting approach!

Reply:
Not sure if it's the best approach but I was getting sick of manually editing and it works!